src/main/java/orangetaxiteam/cocoman/domain/SocialSignSupplier.java

```diff
@@ -0,0 +1,16 @@
+package orangetaxiteam.cocoman.domain;
+
+import java.util.Map;
+
+public class SocialSignSupplier {
+    private final Map<SocialProvider, SocialSignManager> map;
+
+    public SocialSignSupplier(Map<SocialProvider, SocialSignManager> map) {
+        this.map = map;
+    }
+
+    public SocialSignManager supply(SocialProvider socialProvider) {
+        SocialSignManager socialSignManager = this.map.get(socialProvider);
+        return socialSignManager;
```

Comment:
사소한건데 바로 return 해주면 베스트!

```suggestion
        return this.map.get(socialProvider);
```